pkg/schedule/core/cluster_informer.go

```diff
@@ -56,7 +56,9 @@ type CheckerCluster interface {
 // SharedCluster is an aggregate interface that wraps multiple interfaces
 type SharedCluster interface {
 	BasicCluster
+	statistics.StoreStatInformer
```

Comment:
Why do we need it?

Reply:
Because we need `GetStoresLoads` in `GetHotStores` when resolving `/hotspot/hot/stores`.

Reply:
How about using SchedulerCluster